airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/state_converters/abstract_stream_state_converter.py

```diff
@@ -71,12 +72,13 @@ def serialize(self, state: MutableMapping[str, Any], state_type: ConcurrencyComp
         """
         serialized_slices = []
         for stream_slice in state.get("slices", []):
-            serialized_slices.append(
-                {
-                    self.START_KEY: self._to_state_message(stream_slice[self.START_KEY]),
-                    self.END_KEY: self._to_state_message(stream_slice[self.END_KEY]),
-                }
-            )
+            serialized_slice = {
+                self.START_KEY: self._to_state_message(stream_slice[self.START_KEY]),
+                self.END_KEY: self._to_state_message(stream_slice[self.END_KEY]),
+            }
+            if stream_slice.get(self.MOST_RECENT_RECORD_KEY, None):
```

Comment:
tiny nit, but the `None` as the default arg for `get()` is redundant since by default if a key doesn't exist, get returns `None`.